packages/config/config/devices/0x0159/zmnkad1.json

```diff
@@ -0,0 +1,204 @@
+{
+	"manufacturer": "Qubino",
+	"manufacturerId": "0x0159",
+	"label": "ZMNKAD1",
+	"description": "Luxy Smart Switch",
+	"devices": [
+		{
+			"productType": "0x0008",
+			"productId": "0x0052",
+			"zwaveAllianceId": 4079
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "LEDs On/Off",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Brightness Change",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Color Change",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Relay On/Off",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Relay Contact Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No (normally open) output type",
+					"value": 0
+				},
+				{
+					"label": "Nc (normally close) output type",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Device: Auto On Timer",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32525,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Device: Auto Off Timer",
+			"description": "Allowable range: 30–32535 seconds",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32525,
+			"defaultValue": 30,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "12",
+			"label": "Relay: Auto On Timer",
+			"description": "Allowable range: 30–32535 seconds",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32525,
+			"defaultValue": 30,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Relay: Auto Off Timer",
+			"description": "Allowable range: 30–32535 seconds",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32525,
+			"defaultValue": 30,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "30",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Restore Device State After Power Failure"
```

Comment:
We have a different template for this. Probably this one: https://github.com/zwave-js/node-zwave-js/blob/0461e8787c6b154ff3b281d965158c8d38046178/packages/config/config/devices/templates/master_template.json#L367